zim/gui/pageview/__init__.py

```diff
@@ -484,6 +484,33 @@ def image_file_get_dimensions(file_path):
 		raise AssertionError('Could not get size for: %s' % file_path)
 
 
+def _convert_pillow_image_to_pixbuf(image) -> GdkPixbuf.Pixbuf:
```

Comment:
to reduce cognitive complexity.
If you think it's gotten worse, tell me. I'll hide it back.

Reply:
I get your point. It could be useful elsewhere (eg. for plugin authors) as well, maybe worth promoting to be a part of our API? You can leave it here and we can ask Jaap.